pages/expenses.js

```diff
@@ -110,7 +110,7 @@ class ExpensesPage extends React.Component {
             collective={collective}
             LoggedInUser={LoggedInUser}
             displayContributeLink={collective.isActive && collective.host ? true : false}
-            callsToAction={{ hasContact: collective.canContact, hasSubmitExpense: true }}
+            callsToAction={{ hasContact: collective.canContact, hasSubmitExpense: collective.isActive }}
```

Comment:
`isActive` is not the right field for that. This field will be false if the collective is pending an approval from the host, but users should still be able to submit expenses in that case. You may want to look at the `isArchived` field instead.